src/engraving/libmscore/note.cpp

```diff
@@ -1323,8 +1323,8 @@ void Note::draw(mu::draw::Painter* painter) const
         }
         const Staff* st = staff();
         const StaffType* tab = st->staffTypeForElement(this);
-        if (tieBack() && !tab->showBackTied()) {
-            if (chord()->measure()->system() == tieBack()->startNote()->chord()->measure()->system() && el().empty()) {
```

Comment:
this was not right, we may hide note on TAB if it's from the same measure, not system